compile/service.go

```diff
@@ -207,6 +207,11 @@ func (f *FunctionSpec) Link(scope Scope) error {
 	return nil
 }
 
+// HasReturn returns true if this function has a non-void return.
+func (f *FunctionSpec) HasReturn() bool {
+	return f.ResultSpec != nil && f.ResultSpec.ReturnType != nil
```

Comment:
Sure. I wanted to avoid changing that in this change as well. Created https://github.com/thriftrw/thriftrw-go/pull/98